site/templates/application/footer/footer.jet

```diff
@@ -3,22 +3,19 @@
 {{import "./nav.jet"}}
 {{import "./app-badges.jet"}}
 {{import "./sponsor-banner.jet"}}
-{{import "./social-media-buttons.jet"}}
 
-{{block footer()}}
+{{ block footer() }}
```

Comment:
Shannon already won the great spaces-or-no-spaces-in-moustaches-war... don't go changing it back now!